sound/soc/intel/boards/sof_es8336.c

```diff
@@ -28,6 +28,24 @@
 #define SOF_ES8336_SSP_CODEC_MASK		(GENMASK(3, 0))
 
 #define SOF_ES8336_SPEAKERS_EN_GPIO1_QUIRK	BIT(4)
+
```

Comment:
ASoC: Intel: sof_es8336 add support for HDMI_In capture

Adding support for 2 streams of HDMI-In capture
via I2S in CBP_CFP configuration (codec provides bit clock and frame sync)